listener.go

```diff
@@ -132,7 +146,7 @@ func (h *hub) processReqDequeue(req baps3.Message) (resps []*baps3.Message) {
 }
 
 func (h *hub) processReqEnqueue(req baps3.Message) (resps []*baps3.Message) {
-	args := req.AsSlice()[1:]
+	args := req.Args()
 	if len(args) != 4 {
 		return append(resps, baps3.NewMessage(baps3.RsWhat).AddArg("Bad command"))
```

Comment:
Not in the more specific case, eg. a canned 'bad command' message.  In fact, that could be a `var`, but then you'd be able to do weird things like adding arguments to it and it'd mess up global state.